cmake/modules/RootMacros.cmake

```diff
@@ -2407,8 +2407,8 @@ macro(ROOTTEST_COMPILE_MACRO filename)
 
   cmake_path(CONVERT "${CMAKE_CURRENT_BINARY_DIR}" TO_NATIVE_PATH_LIST NATIVE_BINARY_DIR)
   set(root_compile_macro ${CMAKE_COMMAND} -E env
-      ROOT_LIBRARY_PATH="${NATIVE_BINARY_DIR}"
-      ROOT_INCLUDE_PATH="${CMAKE_CURRENT_BINARY_DIR}:${DEFAULT_ROOT_INCLUDE_PATH}"
+      ROOT_LIBRARY_PATH=${NATIVE_BINARY_DIR}
```

Comment:
If I recall correctly, the quote where introduced in attempt to support case where the path had space in them ....